deps/ncrypto/ncrypto.cc

```diff
@@ -1570,11 +1570,66 @@ EVPKeyPointer::ParseKeyResult TryParsePublicKeyInner(
   return EVPKeyPointer::ParseKeyResult(std::move(pkey));
 }
 
-EVPKeyPointer::ParseKeyResult TryParsePublicKeyPEM(
+bool IsASN1Sequence(const unsigned char* data, size_t size,
+                    size_t* data_offset, size_t* data_size) {
+  if (size < 2 || data[0] != 0x30)
```

Comment:
We need to check the size of data to make sure this doesn't crash.

Reply:
That's the `size < 2` check right at the start?

Reply:
duh 😒